### PR TITLE
Feature/run helm test in actions

### PR DESCRIPTION
**Pull request by jonandernovella**
- Run helm tests as part of the integration workflows
- Fix some settings to that the tests pass with the interceptor

### CODE REVIEW

File: sda-svc/test/accession.py [hunk 1]
```diff
@@ -83,16 +83,16 @@
...
 """ % (args.user,
-       args.archivepath,
+       args.inboxpath,
        args.accessionid,
        args.decsha256,
        args.decmd5)
 
 channel.basic_publish(exchange=exchange,
-                      routing_key= 'stableIDs',
+                      routing_key= 'files',
```
Comment (pontus) [inline sda-svc/test/accession.py:95]: I guess this won't work without intercept?

File: dev_tools/scripts/deploy-mq.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@ CEGA_MQ_PASS=$(grep cega_mq_pass sda-deploy-init/config/trace.yml | awk '{print
 
 helm install broker sda-mq \
 --set securityPolicy.create=false,\
+config.verifyPeer=false,\
```
Comment (pontus) [inline dev_tools/scripts/deploy-mq.sh:8]: I think this is fine to merge and get tests enabled, but we really should fix the problem as well.

**Review by pontus — APPROVED** [review]

File: dev_tools/scripts/deploy-mq.sh (hunk 2 above)
Comment (jonandernovella) [reply dev_tools/scripts/deploy-mq.sh:8]: Agreed

File: sda-svc/test/accession.py (hunk 1 above)
Comment (jonandernovella) [reply sda-svc/test/accession.py]: Changed to `accessionIDs`

**Pull request merged**